sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -35,16 +37,16 @@ def test_error_messages_on_wrong_input():
     for score_func in score_funcs:
         expected = ('labels_true and labels_pred must have same size,'
                     ' got 2 and 3')
-        assert_raise_message(ValueError, expected, score_func,
-                             [0, 1], [1, 1, 1])
+        with pytest.raises(ValueError, match=expected):
+            score_func([0, 1], [1, 1, 1])
 
         expected = "labels_true must be 1D: shape is (2"
-        assert_raise_message(ValueError, expected, score_func,
-                             [[0, 1], [1, 0]], [1, 1, 1])
+        with pytest.raises(ValueError, match=re.escape(expected)):
```

Comment:
Better to manually escape with raw strings the message above. (and then remove the `re` import above).

Reply:
> Better to manually escape with raw strings the message above. (and then remove the `re` import above).

But I thought this way is better for readability. What do you think?

Reply:
Well readers won't necessarily know exactly what `re.escape` does without checking the docs. I think explicitly escaping the string simpler to understand.